### PR TITLE
west.yml: hal_stm32: stm32 hci lib update

### PR DESCRIPTION
**Pull request by erwango**
Update hal_stm32 for STM32WB HCI lib update to v1.3.0

cf https://github.com/zephyrproject-rtos/hal_stm32/pull/35

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

### CODE REVIEW

**Comment by MaureenHelm** [comment]
please rebase to fix the merge conflict

**Comment by erwango** [comment]
@MaureenHelm done, thanks!

**Comment by MaureenHelm** [comment]
Please update this to refer to the merged SHA instead of `pull/35/head`

**Comment by erwango** [comment]
> Please update this to refer to the merged SHA instead of pull/35/head

@MaureenHelm Oups, thought it was done already. Fixed

File: boards/arm/nucleo_wb55rg/doc/nucleo_wb55rg.rst [hunk 1]
```diff
@@ -233,8 +233,10 @@ Flashing
 ========
 
 Nucleo WB55RG board includes an ST-LINK/V2-1 embedded debug tool
-interface.  This interface is not yet supported by the openocd version.
-Instead, support can be enabled on pyocd by adding "pack" support with
+interface.  This interface is supported by the openocd version included in the
+Zephyr SDK since v0.11.0.
```
Comment (galak) [inline boards/arm/nucleo_wb55rg/doc/nucleo_wb55rg.rst]: Maybe make this `v0.11.x`

**Review by galak — APPROVED** [review]

**Pull request merged**